spec/defines/vhost_spec.rb

```diff
@@ -741,7 +741,10 @@
           it { is_expected.to contain_concat__fragment('rspec.example.com-redirect') }
           it {
             is_expected.to contain_concat__fragment('rspec.example.com-rewrite').with(
+              content: %r{^\s+RewriteEngine On$},
```

Comment:
You can't test this way (as the action indicates), but you can chain it: `.with(content: ...).with(content: ...)`